application/src/main/java/org/opentripplanner/inspector/vector/geofencing/GeofencingZonesPropertyMapper.java

```diff
@@ -1,38 +1,35 @@
 package org.opentripplanner.inspector.vector.geofencing;
 
-import static org.opentripplanner.street.model.RentalRestrictionExtension.RestrictionType.BUSINESS_AREA_BORDER;
-import static org.opentripplanner.street.model.RentalRestrictionExtension.RestrictionType.NO_DROP_OFF;
-import static org.opentripplanner.street.model.RentalRestrictionExtension.RestrictionType.NO_TRAVERSAL;
-
+import java.util.ArrayList;
 import java.util.Collection;
-import java.util.List;
 import org.opentripplanner.apis.support.mapping.PropertyMapper;
 import org.opentripplanner.inspector.vector.KeyValue;
-import org.opentripplanner.street.model.vertex.Vertex;
+import org.opentripplanner.service.vehiclerental.model.GeofencingZone;
 
 /**
  * A {@link PropertyMapper} for the {@link GeofencingZonesLayerBuilder} for the OTP debug client.
  */
-public class GeofencingZonesPropertyMapper extends PropertyMapper<Vertex> {
+public class GeofencingZonesPropertyMapper extends PropertyMapper<GeofencingZone> {
 
   @Override
-  protected Collection<KeyValue> map(Vertex input) {
-    var ext = input.rentalRestrictions();
+  protected Collection<KeyValue> map(GeofencingZone zone) {
+    var properties = new ArrayList<KeyValue>();
+
+    properties.add(new KeyValue("id", zone.id().toString()));
```

Comment:
There is a helper method `KeyValue.kv` so you don't have to do these conversions here.